gpt_index/readers/file.py

```diff
@@ -31,7 +31,7 @@ def load_data(self, **load_kwargs: Any) -> List[Document]:
         data = ""
         data_list = []
         for input_file in self.input_files:
-            with open(input_file, "r") as f:
+            with open(input_file, "r", errors='replace') as f:
```

Comment:
thanks! could we actually add a param in `__init__` called `errors` that's `replace` by default? then we can just pass through to the `open()` call in `load_data`